api/src/main/resources/workbench-api.yaml

```diff
@@ -1042,7 +1042,29 @@ paths:
       operationId: getWorkspaceResources
       responses:
         200:
-          description: Returns all objects that describe a section of the CDR in a workspace.
+          description: OK
```

Comment:
This is the successful response. The endpoint description (4 lines up) should say what this returns. In my opinion, this should be omitted except in unusual cases, but it is required.